.github/workflows/config.yml

```diff
@@ -0,0 +1,735 @@
+name: celo-blockchain
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+env:
+  # Increment these to force cache rebuilding
+  SYSTEM_CONTRACTS_CACHE_VERSION: 2
+  CHECKOUT_MONOREPO_CACHE_VERSION: 3
```

Comment:
True, although I thought we might not need them at all since we can just manually delete the caches [here](https://github.com/celo-org/celo-blockchain/actions/caches) if needed. Left it as is just to replicate what we had.

Reply:
Ah nice, didn't know about that interface.